libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -213,15 +231,15 @@ class GrayboxPromote extends LitElement {
         repo,
         ref,
         owner,
-        url,
         spData,
+        filePath,
       }) => {
         try {
           const { root, gbRoot, driveId } = spData;
           const mainRepo = repo.replace('-graybox', '');
           const params = {
             adminPageUri: `https://milo.adobe.com/tools/graybox-promote?ref=${ref}&repo=${mainRepo}&owner=${owner}&host=business.adobe.com&project=${mainRepo.toUpperCase()}&referrer=MOCK_REF`,
-            projectExcelPath: getProjectExcelPath(url),
```

Comment:
@arshadparwaiz `host` and `referrer` query param should get removed once the promote API change is made. That is not done yet, correct?